.env

```
@@ -13,3 +13,10 @@ ENV_GARDENER_MAX_NODES=2
 ENV_ISTIO_VERSION=1.11.0
 ENV_K3D_VERSION=v5.4.7
 ENV_GORELEASER_VERSION=v1.23.0
+
+## Default Docker Images
+DEFAULT_FLUENTBIT_EXPORTER_IMAGE="europe-docker.pkg.dev/kyma-project/prod/directory-size-exporter:v20241024-8bc3f6a8"
+DEFAULT_FLUENTBIT_IMAGE="europe-docker.pkg.dev/kyma-project/prod/external/fluent/fluent-bit:3.2.2"
+DEFAULT_OTEL_COLLECTOR_IMAGE="europe-docker.pkg.dev/kyma-project/prod/kyma-otel-collector:0.115.0-main"
+DEFAULT_SELFMONITOR_IMAGE="europe-docker.pkg.dev/kyma-project/prod/tpi/telemetry-self-monitor:3.0.1-857b814"
+DEFAULT_TELEMETRYGEN_IMAGE="ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.115.0"
```

can we rename the variables for test?
eg
```suggestion
DEFAULT_TEST_TELEMETRYGEN_IMAGE="ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.115.0"
```
they would stand out a bit and make them easier to understand

---

renamed